.github/workflows/release.yml

```diff
@@ -132,27 +132,44 @@ jobs:
 
   windows:
     needs: setup
-    runs-on: windows-2022
+    strategy:
+      matrix:
+        include:
+          - runner: windows-2022
+            arch: x86_64
+            zip_name: nvim-win64.zip
+          - runner: windows-11-arm
+            arch: arm64
+            zip_name: nvim-winarm64.zip
+    runs-on: ${{ matrix.runner }}
     steps:
       - uses: actions/checkout@v5
         with:
           # Perform a full checkout #13471
           fetch-depth: 0
       - run: .github/scripts/env.ps1
+        env:
+          BUILD_ARCH: ${{ matrix.arch }}
+      - name: Install Wix
+        run: |
+          Invoke-WebRequest -Uri "https://github.com/wixtoolset/wix3/releases/download/wix3141rtm/wix314-binaries.zip" -OutFile "wix314-binaries.zip"
+          Expand-Archive -Path "wix314-binaries.zip" -DestinationPath "C:\wix"
```

Comment:
you really need to be careful about yaml. I would not be surprised if `"\w"` is treated like an escape. Don't use `\` slashes unless it is absolutely required. 

`\` are rarely required on Windows, despite everyone just automatically assuming that.

Reply:
my suggestion is to use "/"